### ISSUE TITLE
Preserve the zoomed domain of the 2d tally plot between slices

### ISSUE THREAD

**Issue by mkeilman** [radiasoft/sirepo#6235]
The heatmap plot needs a method to get the current domain (`axes.x.scale.domain()` etc.), which we can set when the slice position changes.

**Comment by moellep** [comment]
I think this can be as easy as passing a "preserveZoom" argument to reload, and letting the heatmap handle it:

```javascript
            if (! axes.x.values || ! json.preserveZoom) {
                $.each(axes, function(dim, axis) {
                    axis.values = plotting.linearlySpacedArray(json[dim + '_range'][0], json[dim + '_range'][1], jso\
n[dim + '_range'][2]);
                    axis.updateLabel(json[dim + '_label'], select);
                    axis.scale.domain(getRange(axis.values));
                });
            }
```

**Comment by moellep** [comment]
Fixed with #6331
